poisson2d/README.md

```diff
@@ -0,0 +1,31 @@
+## Poisson solver in various languages
+
+This solver uses a simple Jacobi iteration to solve a Poisson equation. For details, see, for example,
+"Computational Physics" by Mark Newman, Chap 9. Compiler commands were:
+
+```gfortran sourcefile.f95 -Ofast -o program```
+
+```gcc sourcefile.c -Ofast -o program```
+
+For Cython module (import it to run):
+
+```python setup.py build_ext --inplace```
+
+And the timing results (the grid is MxM):
+
+| Language            | M=100 (seconds) | M=200 (seconds)       | M=300 (seconds)        |
```

Comment:
```suggestion
And the timing results in seconds (the grid is MxM):

| Language            | M=100           | M=200                 | M=300                  |
```

Reply:
Yup, I made a change somewhat like this.